includes/definitions.inc.php

```diff
@@ -1534,13 +1543,6 @@
 $config['os'][$os]['over'][0]['graph'] = 'device_toner';
 $config['os'][$os]['over'][0]['text']  = 'Toner';
 
-$os = 'richoh';
```

Comment:
Unfortunately you've removed the richoh code here :(

If you can add this back in pls.